sonar-ws/src/main/java/org/sonarqube/ws/client/WsResponse.java

```diff
@@ -57,5 +58,7 @@
   Reader contentReader();
 
   String content();
+  
+  void close();
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add the "@Override" annotation above this method signature [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1161)